### PR TITLE
Native smoke test for webclient

### PR DESCRIPTION
**Pull request by danielkec**
Signed-off-by: Daniel Kec <daniel.kec@oracle.com>

### CODE REVIEW

File: tests/integration/native-image/se-1/src/main/java/io/helidon/tests/integration/nativeimage/se1/Se1Main.java [hunk 1]
```diff
@@ -40,6 +43,8 @@
  * Main class of this integration test.
  */
 public final class Se1Main {
+    static LazyValue<WebServer> server;
```
Comment (tomas-langer) [inline tests/integration/native-image/se-1/src/main/java/io/helidon/tests/integration/nativeimage/se1/Se1Main.java:46]: Why?

File: tests/integration/native-image/se-1/src/main/resources/application.yaml [hunk 2]
```diff
@@ -23,3 +23,16 @@ server:
...
+  api-version: 2
+  # mocked zipkin
+  path: "/zipkin/api/v2/spans"
+
+client:
+  follow-redirects: true
+  max-redirects: 5
+  services:
+    tracing:
+    security:
```
Comment (tomas-langer) [inline tests/integration/native-image/se-1/src/main/resources/application.yaml:38]: There is no configuration for security. How does the security feature work here?

File: tests/integration/native-image/se-1/src/main/java/io/helidon/tests/integration/nativeimage/se1/Se1Main.java (hunk 1 above)
Comment (danielkec) [reply tests/integration/native-image/se-1/src/main/java/io/helidon/tests/integration/nativeimage/se1/Se1Main.java]: Rather obscure way how to get current webserver port inside the service, removed and simplified

File: tests/integration/native-image/se-1/src/main/resources/application.yaml (hunk 2 above)
Comment (danielkec) [reply tests/integration/native-image/se-1/src/main/resources/application.yaml]: Copy paste error, sorry

**Pull request merged**